src/gluonts/mx/model/estimator.py

```diff
@@ -45,9 +45,17 @@ class GluonEstimator(Estimator):
 
     @validated()
     def __init__(
-        self, trainer: Trainer, lead_time: int = 0, dtype: DType = np.float32
+        self,
+        trainer: Trainer,
+        batch_size: int = 32,
```

Comment:
Maybe we can ensure that estimators written by someone else doesn't accidentally calls the super with the old signature and sets the batch_size with the `lead_time` value. Its a bit contrived, but perhaps we can ensure that the caller uses kwargs or something?